public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -75,28 +63,28 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         address quoteAddress;
     }
 
-    event ReportVerified(BasicReport indexed report);
-    event PriceUpdate(int192 price);
+    event PriceUpdate(int192 indexed price);
 
-    address public immutable FEE_ADDRESS;
+    IVerifierProxy public verifier;
+
+    address public FEE_ADDRESS;
     string public constant STRING_DATASTREAMS_FEEDLABEL = "feedIDs";
     string public constant STRING_DATASTREAMS_QUERYLABEL = "timestamp";
-    string[] public feedsHex = [
-        "0x00023496426b520583ae20a66d80484e0fc18544866a5b0bfee15ec771963274"
+    string[] public feedIds = [
+        "0x00029584363bcf642315133c335b3646513c20f049602fc7d933be0d3f6360d3" // Ex. Basic ETH/USD price report
     ];
 
-    constructor(address _feeAddress, address _verifier) {
-        verifier = IVerifierProxy(_verifier); //0xea9B98Be000FBEA7f6e88D08ebe70EbaAD10224c
-        FEE_ADDRESS = _feeAddress; // 0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3 (WETH)
+    constructor(address _verifier) {
+        verifier = IVerifierProxy(_verifier);
     }
 
     function checkLog(
         Log calldata log,
         bytes memory
-    ) external view returns (bool upkeepNeeded, bytes memory performData) {
+    ) external returns (bool upkeepNeeded, bytes memory performData) {
         revert StreamsLookup(
```

Comment:
Just a thought for the docs, may not be appropriate in-line
but this revert pattern is pretty weird for those who dont know the rationale.  In the docs, it may be helpful to point to EIP3668 (https://eips.ethereum.org/EIPS/eip-3668#rationale) the rationale section for a background on this pattern?

Reply:
Done